src/components/navigation/Navbar.tsx

```diff
@@ -1,12 +1,33 @@
 import React from "react";
 import { Link } from "react-router-dom";
+import "./Navbar.scss";
+import { useState } from 'react'
 
 export const Navbar: React.FunctionComponent = () => {
+
+  const [isExpanded, setIsExpanded] = useState(false)
+
+  function toggleMenu(){
+    setIsExpanded(current => !current)
+  }
+
   return <nav>
-    <ul>
-      <li><Link to="/">Home</Link></li>
-      <li><Link to="/rovers">Rovers</Link></li>
-      <li><Link to="/quiz">Quiz</Link></li>
-    </ul>
+
+    <div className="navbar">
+      <button onClick = {() => toggleMenu()}>
+      
+        <span className = {`navbar__spanone ${isExpanded ? "burgerexpanded" : "burgercollapsed"}`}></span>
+        <span className = {`navbar__spantwo ${isExpanded ? "burgerexpanded" : "burgercollapsed"}`}></span>
+        <span className = {`navbar__spanthree ${isExpanded ? "burgerexpanded" : "burgercollapsed"}`}></span>
+
+      </button>
+      
```

Comment:
A few unnecessary blank lines:
```suggestion
    <div className="navbar">
      <button onClick = {() => toggleMenu()}>
        <span className = {`navbar__spanone ${isExpanded ? "burgerexpanded" : "burgercollapsed"}`}></span>
        <span className = {`navbar__spantwo ${isExpanded ? "burgerexpanded" : "burgercollapsed"}`}></span>
        <span className = {`navbar__spanthree ${isExpanded ? "burgerexpanded" : "burgercollapsed"}`}></span>
      </button>
```

Reply:
done, thanks